javascript/javascript1/week4/fibnum.js

```diff
@@ -0,0 +1,15 @@
+let n1 = 0;
+let n2 = 1;
+
+function fib(n) {
+   let fibNum = [n1, n2];
+   for (let i = 2; i <= n; i++) {
+      fibNum.push(fibNum[i - 1] + fibNum[i - 2]);
+   }
+   return fibNum[n - 1];
+}
+
+console.log(fib(5)) //3
+console.log(fib(10)) //34
+
```

Comment:
This function was the first most commonly taken approach. Hope you've seen Rasmus's code for the 3rd approach, which is most optimal on processing memory - https://github.com/HackYourFuture-CPH/JavaScript/blob/team28/javascript1/week4/inclass/fibo.js